### ISSUE TITLE
Sync: PHP Notice in Sync\Functions

### ISSUE THREAD

**Issue by rilwis** [Automattic/jetpack#14276]
When I access the WP Admin on my localhost, I see this notice:

```
Notice: Automattic\Jetpack\Sync\Functions was called incorrectly. Not all plugins have loaded yet but we requested the class Automattic\Jetpack\Sync\Functions Please see Debugging in WordPress for more information. (This message was added in version dev-retry/phpcs-changed.) in /path/to/my/local/wp-includes/functions.php on line 4986
```

![Screen Shot 2019-12-25 at 4 45 36 PM](https://user-images.githubusercontent.com/668274/71441890-127b2900-2736-11ea-93a5-476cd01547d9.png)

My environment:

- Mac OS with built-in Apache + PHP
- WP 5.3.2

**Comment by kbrown9** [comment]
Thanks for reporting this!

I think that you’re experiencing the same issue that was reported [here](https://github.com/Automattic/jetpack/issues/13048#issuecomment-525619758) and fixed in PR #14251. That PR is scheduled to be included in the next release of Jetpack, version 8.1.

If you can get the stack trace for the PHP notice, we can verify the cause.

**Comment by jeherve** [comment]
Closing for now since it should be fixed in the next Jetpack release. Feel free to comment again if this keeps happening after the release!
